packages/mobile_app/lib/screens/registration/screen.dart

```diff
@@ -0,0 +1,226 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mutable_wrappers/mutable_wrappers.dart';
+import 'package:language/language.dart';
+import 'package:app/theme/app_colors.dart';
+import 'package:app/blocs/authentication.dart';
+import 'package:app/components/userinfo_field.dart';
+import 'package:app/components/userinfo.dart';
+import 'package:app/components/gradient_flatbutton.dart';
+import 'package:app/components/userinfo_picker.dart';
+import 'package:user_model/user_model.dart';
+import 'accept_terms.dart';
+
+class RegistrationScreen extends StatelessWidget {
+  final name = MutableString("");
+  final month = MutableString("");
+  final day = MutableString("");
+  final year = MutableString("");
+  final sex = MutableInteger(0);
+  final indexToSex = {
+    0: lang.notSelectedSex,
+    1: lang.male,
+    2: lang.female,
+    3: lang.other,
+  };
+  final country = MutableString("");
+  final place = MutableString("");
+  final termsAccepted = MutableBool(false);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.transparent,
+      body: SafeArea(
+        child: ListView(
```

Comment:
I guess, it was somewhere in the chat, but I cannot recall Can you please remind, why are you using ListView here?

Reply:
Because our screen is too big and needs scrolling. Also, without it it can crush in small sized screen devices.